### ISSUE TITLE
Do not support multi-GPU training

### ISSUE THREAD

**Issue by sjtuxcx** [princewang1994/RFCN_CoupleNet.pytorch#8]
Hi, I found this code do not support multi-GPU training. Problems will be produced as follows:
Argument is located on different GPUs when doing loss.backward().  It is caused by psroi_pooling module. So the psroi_pooling code may need to be modified.

**Comment by sjtuxcx** [comment]
I solve it. Thanks.
